assessments/KRAssessment/README.md

```diff
@@ -0,0 +1,61 @@
+# Assessment Details
```

Comment:
can you update the readme to match the others in the repo?

Reply:
Although I have resolved all the other issues, I don't know what I would need to do here. Can you elaborate? @jalvord1